### PR TITLE
Add Design QA

### PR DESCRIPTION
**Pull request by martins87**
## Issue
https://app.clickup.com/t/2j8fe12

## Description
Changes added to Pablo project, now to Composable monorepo

Comments by Design Team addressed:
1. Header balance to 2 point decimal. Wallet box responsive to wallet name
2. Swap page to have default tokens selected
3. Hover state for unselected buttons under 'Slippage tolerance'
4. Empty state for Auctions tab similar to bond page 
5. Select token component: bottom space on dropdown equal to the top. Entire component centered to screen
6. Token names on swap details to All Caps
7. Dropdown states for Token 1 and token 2 set to input text placeholder: "Select a token"
8. Warning box: info icon changed to orange
9. Pool fees input fixed
10. Confirm new pool step: amount of tokens shown (before just token label)
11. Select token component: Bottom space on dropdown set to equal to the top
12. Checkboxes roundness increased

## Checklist

- [ ] I have updated the cargo docs to reflect changes made by this PR _(if applicable)_
- [ ] I have updated the `book/` to reflect changes made by this PR _(if applicable)_

@KaiserKarel

### CODE REVIEW

File: frontend/ui-pablo/app/components/Organisms/Wallet/PolkadotConnect.tsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ const Status = () => {
               value: asset.assetId,
               label:
               balance.lte(1000)
-                  ? balance.toString()
+                  ? balance.toFixed(2).toString()
```
Comment (rodinaleksandr0219) [inline frontend/ui-pablo/app/components/Organisms/Wallet/PolkadotConnect.tsx:52]: @martins87 you don't need to use toString for toFixed. toFixed returns string.

**Comment by KaiserKarel** [comment]
Could you elaborate a bit more in the PR itself, describing the changes? Outsides cannot access our clickup issue ticket.

File: frontend/ui-pablo/app/components/Organisms/Wallet/PolkadotConnect.tsx (hunk 1 above)
Comment (martins87) [reply frontend/ui-pablo/app/components/Organisms/Wallet/PolkadotConnect.tsx]: Fixed

**Comment by martins87** [comment]
> Could you elaborate a bit more in the PR itself, describing the changes? Outsides cannot access our clickup issue ticket.

**Review by easteregg — APPROVED** [review]

**Pull request closed without merging**